src/component/chrome/chrome-container.tsx

```diff
@@ -24,13 +27,17 @@ export class ChromeContainer extends React.Component {
 		}
 
 		return (
-			<Chrome
-				leftVisible={!!previousPage}
-				rightVisible={!!nextPage}
-				onLeftClick={() => (previousPage ? store.openPage(previousPage.getId()) : undefined)}
-				onRightClick={() => (nextPage ? store.openPage(nextPage.getId()) : undefined)}
-				title={page ? page.getName() : undefined}
-			>
+			<Chrome>
+				<OverviewSwitchContainer />
+				<ViewSwitch
+					fontSize={Size.M}
+					justify="center"
+					leftVisible={!!previousPage}
```

Comment:
please refactror this too to use `!!` but `Boolean()`. The `!!` is pretty easy to oversee but `Boolean()` is easy to see. Afaik that is also something that was agreed on for this project. Correct me if I'm wrong

Reply:
Done ✅